restlib/restlib_test.go

```diff
@@ -65,6 +66,19 @@ func TestUnmarshalEmptyBodyOK(t *testing.T) {
 	require.Nil(t, result.Response)
 }
 
+func TestUnmarshalImageContentType(t *testing.T) {
+	header := map[string][]string{
+		"Content-Type": {"image/png"},
+	}
+	result, err := unmarshal(&http.Response{Header: header}, []byte(okJSON), &OkType{})
```

Comment:
The logic is fine but can you change the example from: 
```
[]byte(okJSON)
``` 

into something like 
```
var image = [...]byte { 0, 1 }
```

I was a bit confused when the test is for an image response and the response contained a json string.